dgen_os/python/input_data_functions.py

```diff
@@ -373,31 +408,44 @@ def function(df):
 
 
 #%%
-def import_agent_file(scenario_settings, con, cur, engine, model_settings, agent_file_status, input_name):
+def import_agent_file(scenario_settings, con, model_settings, agent_file_status, input_name):
     """
     Generates new agents or uses pre-generated agents from provided .pkl file
     
     Parameters
     ----------
-    scenario_settings : 'SQL schema'
-        Schema of the scenario settings
-    con : 'SQL connection'
+    scenario_settings : :class: `settings.ScenarioSettings`
+        Custom object in which to look for the scenario settings
+
+    con : :class: `psycopg2.connection`
         SQL connection to connect to database
-    cur : 'SQL cursor'
-        Cursor
-    engine : 'SQL engine'
-        SQL engine to intepret SQL query
-    model_settings : 'object'
+
+    model_settings : :class: `settings.ModelSettings`
         Model settings that apply to all scenarios
-    agent_file_status : 'attribute'
-        Attribute that describes whether to use pre-generated agent file or create new    
-    input_name : 'string'
+
+    agent_file_status : `string`
```

Comment:
```suggestion
    agent_file_status : str
```

Reply:
This variable may be removed due to the above comment about the pre-generated agents